recipes/ocp/meta.yaml

```diff
@@ -0,0 +1,70 @@
+{% set name = "ocp" %}
+{% set version = "7.5.3" %}
+{% set occt_ver = "7.5.3" %}
+{% set ocp_tweak = "0" %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  # currently only used for the license file
+  - url: https://github.com/CadQuery/OCP/archive/refs/tags/{{ occt_ver }}.{{ ocp_tweak }}.tar.gz
+    folder: ocp
+    md5: 15c6c85c616462d5b70356549a6446f1
```

Comment:
these all need sha256sums... md5 is basically broken at this point.

Reply:
and if you _really_ need just the license, you can explicitly grab it, and specify an `fn` instead of `folder`... however, it can have some annoying local cache issues, so i sometimes end up with:

```yaml
- fn: the-package-{{ version }}-LICENSE.txt
  url: http://foo/bar/LICENSE.txt
  sha256sum: ...
```